deps/jerry/jerry-core/include/jerryscript-core.h

```diff
@@ -537,6 +537,8 @@ bool jerry_enable_cpu_profiling (void);
 bool jerry_start_cpu_profiling (const char *path, double duration);
 bool jerry_stop_cpu_profiling (void);
 
+void jerry_heap_profiler_take_snapshot (const char *path);
```

Comment:
`jerry_start_cpu_profiling` doesn't own a prefix like `cpu_profiler`, how about the shorter name `jerry_take_snapshot` here?